### DISCUSSION TITLE
Python Data Source

### DISCUSSION THREAD

**Discussion by Anthony-WematchLive** [getredash/redash#6649] [Q&A] [answered]
Hello,

I try to use the Data Source python directly to Redash, by following this setup https://redash.io/help/data-sources/querying/python.
But I didn't succeed, after the env update to see Python as a new data source in Redash.
Anyone had already the same problem ?

Thanks all for your help,
Anthony

**Comment by dspv** [comment]
I got the same problem:
1. I added variable /opt/redash/env -- didn't help
2. I added variable to services.redash.environment -- didn't help

**Reply by gaecoli** [reply, marked as answer]
You should add `REDASH_ADDITIONAL_QUERY_RUNNERS=redash.query_runner.python` to your Redash env file.
Maybe `.env` or `/opt/redash/env`.

**Reply by gaecoli** [reply]
If you add this env, you have to restart Redash, it will work.
